TLM/TLM/Manager/Impl/AdvancedParkingManager.cs

```diff
@@ -2902,7 +2901,7 @@ bool citizenDebug
                                                                 out Vector3 parkPos,
                                                                 out Quaternion parkRot,
                                                                 out float parkOffset) {
-            if (!CustomPassengerCarAI.FindParkingSpace(isElectric,
+            if (!Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection.FindParkingSpace(isElectric,
```

Comment:
consider a const for `Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection`